solution.rb

```diff
@@ -0,0 +1,29 @@
+TEXT = File.read('speech.txt')
+
+def find_words text
+  words_regex = /\w+[a-z']*\w+/
```

Comment:
It'd probably be a good idea to encapsulate these methods in a class as well, no?